README.md

```diff
@@ -93,6 +93,22 @@ Diagrams are generated "automatically" by ???
 
 ## User Guide
 
+###Database Management
```

Comment:
also, move this up into the "Development" subsection of the readme

Reply:
Done, that makes more sense.